KeyboardSpy/KeyboardSpy/KeyboardSpy.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+
+namespace KeyboardSpy
+{
+    public static class KeyboardSpy
+    {
+        private const int WH_KEYBOARD_LL = 13;
+        private const int WM_KEYDOWN = 0x0100;
+
+        private static LowLevelKeyboardProc _proc = HookCallback;
+        private static IntPtr _hookID = IntPtr.Zero;
+
+        private delegate IntPtr LowLevelKeyboardProc(int nCode, IntPtr wParam, IntPtr lParam);
```

Comment:
`delegate` is type declaration, so please move it to the end of the class (prepend with 3 empty lines) or to separate file